builder_data.yaml

```diff
@@ -50,10 +50,11 @@ apt_repo_command: |
   keychain -k mine;
 yum_repo_command: |
   for repodir in $(find "__REPO_PATH__" -mindepth 2 -name "*.rpm" | xargs -I {} dirname {} | uniq) ; do
-    [ -d "${repodir}" ] || continue;
-    echo "Generating repodata for ${repodir}..."
-    sudo createrepo --checksum=sha --database --update "${repodir}";
-    echo "Finished generating repodata for ${repodir}..."
+    [ -d "${repodir}" ] || continue &&
+    echo "Generating and signing repodata for ${repodir} with __GPG_KEY__..." &&
+    sudo createrepo --checksum=sha --database --update "${repodir}" &&
+    gpg --armor --detach-sign -u __GPG_KEY__ "${repodir}/repodata/repomd.xml" &&
+    gpg --export --armour __GPG_KEY__ > "${repodir}/repodata/repomd.xml.key"
```

Comment:
This isn't sufficient for sles 11, I'm still digging into why that is. When the key is created this was and then imported, the package signature verification fails with a bad signature.